packages/webdriverio/src/utils/index.ts

```diff
@@ -116,13 +116,6 @@ export function transformToCharString (value: any, translateToUnicode = true) {
         } else if (typeof val === 'number') {
             const entry = `${val}`.split('')
             ret.push(...entry)
-        } else if (val && typeof val === 'object') {
-            try {
-                ret.push(...JSON.stringify(val).split(''))
-            } catch (e) { /* ignore */ }
-        } else if (typeof val === 'boolean') {
-            const entry = val ? 'true'.split('') : 'false'.split('')
-            ret.push(...entry)
```

Comment:
mhm 🤔 I am wondering if anyone did use this. I would keep it around to not make this a breaking change for folks that do use this.

Reply:
This was only used by `addValue` in the codebase, hence I felt confident it could be removed. Do you strongly feel we should keep this around?

It's not part of our API and I therefor I feel that we shouldn't take into account that someone might have used some of our internal utils.

Reply:
Also, this change in itself is already a breaking one by changing the allowed types/values

Reply:
How about we add a deprecation warning, via:

```js
log.warn("...")
```

And remove it eventually at some point later. While it technically is a breaking change I assume that not the majority of users use TS. Changing this functionality actually does impact everyone.